### ISSUE TITLE
Respawning features missing in 1.16

### ISSUE THREAD

**Issue by k1r0s** [TheIllusiveC4/CorpseComplex#73]
https://github.com/TheIllusiveC4/CorpseComplex/wiki/Respawning

Documentation must point out that these features are yet to be implemented.

**Comment by k1r0s** [comment]
https://github.com/TheIllusiveC4/CorpseComplex/issues/44  related

**Comment by SamayaNL** [comment]
Yeah I was so confused when I couldnt find it. So is it a bug and fixed or not, or as yet to be implemented?
